specs/commandline/plugin.md

```diff
@@ -36,15 +36,55 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install --name <plugin name> --plugin-package <plugin package> [flags]
```

Comment:
How about
```suggestion
  notation plugin install [--name <plugin name>] [flags] <plugin package> 
```

Reply:
Later we can have
```console
$ notation plugin install notation-example-foo.tgz
The NEW plugin `example-foo` will be installed.
Do you want to continue? [y/N] y
The plugin `example-foo` is successfully installed.
$ notation plugin install --name example-bar notation-example-bar.tgz
The plugin `example-bar` is successfully installed.
$ notation plugin install --name example-hello notation-example-foo.tgz
Error: notation-example-foo.tgz: example-hello not found
```

Reply:
I like it, that makes a lot of sense. +100